.travis.yml

```diff
@@ -10,6 +11,17 @@ env:
   - FEDORA_VERSION=3.6.2
   - FEDORA_VERSION=3.7.0
   - FEDORA_VERSION=3.7.1
+before_install:
+  - wget https://github.com/fcrepo3/fcrepo/releases/download/v3.8.1/install-f3-deps.zip
+  - unzip install-f3-deps.zip
+  - cd install-f3-deps/lib
+  - wget http://alpha.library.yorku.ca/mulgara-core-2.1.12.jar
+  - wget http://alpha.library.yorku.ca/mulgara-core-2.1.12.pom
+  - rm mulgara-core-2.1.13.jar mulgara-core-2.1.13.pom
```

Comment:
Any particular reason to replace 13 with 12?... Just seems kind of weird.
Also, this whole procedure is kind of binding the build to 3.8.1 explicitly, yeah?